superset-frontend/src/SqlLab/components/SaveDatasetModal/SaveDatasetModal.test.tsx

```diff
@@ -55,9 +63,732 @@ const mockedProps = {
   datasource: testQuery,
 };
 
-describe('SaveDatasetModal RTL', () => {
-  it('renders a "Save as new" field', () => {
-    render(<SaveDatasetModal {...mockedProps} />, { useRedux: true });
+const testDatasetList = {
```

Comment:
can we put this in a separate fixture file?

Reply:
Yeah, once I'm done fixing this test file I'll be cleaning it up.